### PR TITLE
Fixed Sanitization and escaping

### PR DESCRIPTION
**Pull request by ManiruzzamanAkash**
[notice] Promotional Black Friday Notice Added
[fix] Fixed Sanitisation for Unsanitized echo and print.
[fix] Removed Localhost related dummy datas.
[fix] Removed some unused .zip files.
[fix] Removed External Image Links and added inside plugin.
[fix] Removed external unused CDN links.
[fix] Updated Chart JS out of date libraries.
[fix] Fixed Some phpcs overall plugin.
[fix] Remove unnecessary redundant sanitizatoin.

### CODE REVIEW

File: core/User_Profile/Profile_Update.php [hunk 1]
```diff
@@ -80,18 +80,18 @@ protected function capbility_form( $user ) {
         <table class="form-table">
             <tbody>
                 <tr>
-                    <th><?php _e( 'Capability', 'pm-pro' ); ?> </th>
+                    <th><?php esc_html_e( 'Capability', 'pm-pro' ); ?> </th>
```
Comment (tanmayjay) [inline core/User_Profile/Profile_Update.php:83]: The domain here is supposed to be `wedevs-project-manager`

File: views/emails/html/new-comment.php [hunk 2]
```diff
@@ -89,10 +86,10 @@
                         <tr>
                             <td>
                                 <p style="font-size: 16px; line-height: 30px; margin: 0; color: #7E8690;">
-                                    <span><?php _e( 'Comment', 'wedevs-project-manager' ); ?></span>
+                                    <span><?php esc_html_e( 'Comment', 'wedevs-project-manager' ); ?></span>
                                 </p>
                                 <div style="padding: 5px 15px; background: #edeef45e; border: 1px solid #f2f2f2; border-radius: 5px; margin-bottom: 30px; line-height: 26px; margin-top: 10px;">
-                                    <?php echo $content; ?>
+                                    Test Comment: <?php echo wp_kses_data( $content ); ?>
```
Comment (tanmayjay) [inline views/emails/html/new-comment.php:92]: Escape the string here and make it translatable as well.
`<?php echo esc_html__( 'Test Comment: ', 'wedevs-project-manager' ) . wp_kses_data( $content ); ?>`

File: views/emails/html/new-message.php [hunk 3]
```diff
@@ -121,37 +118,34 @@
...
+                                                margin-right: 4px;
+                                                text-decoration:none;
+                                                height: 33px;
+                                                width: 33px;
+                                                border-radius: 50%;"
 
-                                            src="<?php echo esc_url( $assign->avatar_url ); ?>" 
-                                            alt="<?php echo $assign->display_name; ?>" title="<?php echo $assign->display_name; ?>" width="33" height="33" 
+                                            src="<?php echo esc_url( $assign->avatar_url ); ?>"
+                                            alt="<?php echo esc_html( $assign->display_name ); ?>" title="<?php echo esc_html( $assign->display_name ); ?>" width="33" height="33"
```
Comment (tanmayjay) [inline views/emails/html/new-message.php:139]: Use `esc_attr` as these values are being used as the values of attributes.
`alt="<?php echo esc_attr( $assign->display_name ); ?>" title="<?php echo esc_attr( $assign->display_name ); ?>" width="33" height="33"`

File: views/emails/html/new-task.php [hunk 4]
```diff
@@ -4,82 +4,79 @@
 <head>
     <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
-    <title>Your Message Subject or Title</title>
+    <title><?php echo esc_html_e( 'Your Message Subject or Title', 'wedevs-project-manager' ); ?></title>
```
Comment (tanmayjay) [inline views/emails/html/new-task.php:7]: `echo` is not needed while using `esc_html_e`

File: views/emails/html/update-project.php [hunk 5]
```diff
@@ -75,22 +73,18 @@
...
-                                                '%s "%s" %s %s', 
-                                                __( 'The project', 'wedevs-project-manager'  ),
-                                                $title,
-                                                __( 'has been updated by', 'wedevs-project-manager' ),
-                                                $updater['data']['display_name']
+                                            printf(
+                                                '%s "%s" %s %s',
+                                                esc_html_e( 'The project', 'wedevs-project-manager'  ),
+                                                esc_html( $title ),
+                                                esc_html_e( 'has been updated by', 'wedevs-project-manager' ),
```
Comment (tanmayjay) [inline views/emails/html/update-project.php:80]: `_e` is not needed inside `printf`. use `__` instead.
However the sentece should be intact to make a proper and flexible translation.
```
        printf(
            esc_html__( 'The project "%1$s" has been updated by %2$s', 'wedevs-project-manager'  ),
            $title,
            $updater['data']['display_name']
        );
```

File: views/emails/html/update-task.php [hunk 6]
```diff
@@ -10,93 +10,92 @@
...
                     </tr>
 
                     <tr>
                         <td colspan="2">
                             &nbsp;
-                            <p style="font-size: 16px; line-height: 30px; margin: 0; color: #7E8690;"><?php _e( 'Description', 'wedevs-project-manager' ); ?></p>
+                            <p style="font-size: 16px; line-height: 30px; margin: 0; color: #7E8690;"><?php esc_html_e( 'Description', 'wedevs-project-manager' ); ?></p>
                             <div style="padding: 5px 15px; background: #edeef45e; border: 1px solid #f2f2f2; border-radius: 5px; margin-bottom: 30px; line-height: 26px; margin-top: 10px;">
-                                <?php echo empty( $description['html'] ) ? __( 'No description found!', 'wedevs-project-manager' ) : $description['html']; ?>
+                                Test Update task: <?php echo empty( $description['html'] ) ? esc_html_e( 'No description found!', 'wedevs-project-manager' ) : wp_kses_data( $description['html'] ); ?>
```
Comment (tanmayjay) [inline views/emails/html/update-task.php:90]: Is this for test purpose? Found a similar thing above. If not, change the line as below:
```
<?php echo esc_html__( 'Test Update task: ', 'wedevs-project-manager' ) . empty( $description['html'] ) ? esc_html__( 'No description found!', 'wedevs-project-manager' ) : wp_kses_data( $description['html'] ); ?>
```

**Review by tanmayjay — CHANGES_REQUESTED** [review]

File: core/Promotions/Offers.php [hunk 7]
```diff
@@ -48,6 +48,11 @@ public function promotional_offer() {
             return;
         }
 
+        // Check if inside the wp-project-manager page
+        if ( ! isset( $_GET['page'] ) || 'pm_projects' !== $_GET['page'] ) { // phpcs:disable WordPress.Security.NonceVerification
```
[the remark this answers is not in the document]
Comment (ManiruzzamanAkash) [reply core/Promotions/Offers.php:52]: Is that necessary in isset checking? I don't find any sanitization in if with isset block in WordPress core even.

**Review by tanmayjay — APPROVED** [review]

**Pull request merged**